<!DOCTYPE html><html lang="en" ><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><meta name="generator" content="Jekyll v4.2.2" /><meta property="og:title" content="Transcriptome I - Introduction to Transcriptome" /><meta property="og:locale" content="en" /><meta name="description" content="본 post는 국가생명연구자원정보센터(KOBIC) 주관 이화여자대학교 생명과학과 이상혁 교수님의 전사체 데이터 분석를 정리한 내용입니다." /><meta property="og:description" content="본 post는 국가생명연구자원정보센터(KOBIC) 주관 이화여자대학교 생명과학과 이상혁 교수님의 전사체 데이터 분석를 정리한 내용입니다." /><link rel="canonical" href="/posts/Transcriptome1/" /><meta property="og:url" content="/posts/Transcriptome1/" /><meta property="og:site_name" content="Hubert Effect" /><meta property="og:type" content="article" /><meta property="article:published_time" content="2022-11-10T07:58:52+09:00" /><meta name="twitter:card" content="summary" /><meta property="twitter:title" content="Transcriptome I - Introduction to Transcriptome" /><meta name="twitter:site" content="@twitter_username" /> <script type="application/ld+json"> {"@context":"https://schema.org","@type":"BlogPosting","dateModified":"2022-11-22T15:45:27+09:00","datePublished":"2022-11-10T07:58:52+09:00","description":"본 post는 국가생명연구자원정보센터(KOBIC) 주관 이화여자대학교 생명과학과 이상혁 교수님의 전사체 데이터 분석를 정리한 내용입니다.","headline":"Transcriptome I - Introduction to Transcriptome","mainEntityOfPage":{"@type":"WebPage","@id":"/posts/Transcriptome1/"},"url":"/posts/Transcriptome1/"}</script><title>Transcriptome I - Introduction to Transcriptome | Hubert Effect</title><link rel="apple-touch-icon" sizes="180x180" href="/assets/img/favicons/apple-touch-icon.png"><link rel="icon" type="image/png" sizes="32x32" href="/assets/img/favicons/favicon-32x32.png"><link rel="icon" type="image/png" sizes="16x16" href="/assets/img/favicons/favicon-16x16.png"><link rel="manifest" href="/assets/img/favicons/site.webmanifest"><link rel="shortcut icon" href="/assets/img/favicons/favicon.ico"><meta name="apple-mobile-web-app-title" content="Hubert Effect"><meta name="application-name" content="Hubert Effect"><meta name="msapplication-TileColor" content="#da532c"><meta name="msapplication-config" content="/assets/img/favicons/browserconfig.xml"><meta name="theme-color" content="#ffffff"><link rel="preconnect" href="https://fonts.googleapis.com" ><link rel="dns-prefetch" href="https://fonts.googleapis.com" ><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link rel="dns-prefetch" href="https://fonts.gstatic.com" crossorigin><link rel="preconnect" href="https://fonts.googleapis.com" ><link rel="dns-prefetch" href="https://fonts.googleapis.com" ><link rel="preconnect" href="https://cdn.jsdelivr.net" ><link rel="dns-prefetch" href="https://cdn.jsdelivr.net" ><link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Lato&family=Source+Sans+Pro:wght@400;600;700;900&display=swap"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4/dist/css/bootstrap.min.css"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.11.2/css/all.min.css"><link rel="stylesheet" href="/assets/css/style.css"><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@1.0.1/dist/bootstrap-toc.min.css"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/magnific-popup@1/dist/magnific-popup.min.css"> <script src="https://cdn.jsdelivr.net/npm/jquery@3/dist/jquery.min.js"></script> <script type="text/javascript"> class ModeToggle { static get MODE_KEY() { return "mode"; } static get MODE_ATTR() { return "data-mode"; } static get DARK_MODE() { return "dark"; } static get LIGHT_MODE() { return "light"; } static get ID() { return "mode-toggle"; } constructor() { if (this.hasMode) { if (this.isDarkMode) { if (!this.isSysDarkPrefer) { this.setDark(); } } else { if (this.isSysDarkPrefer) { this.setLight(); } } } let self = this; /* always follow the system prefers */ this.sysDarkPrefers.addEventListener("change", () => { if (self.hasMode) { if (self.isDarkMode) { if (!self.isSysDarkPrefer) { self.setDark(); } } else { if (self.isSysDarkPrefer) { self.setLight(); } } self.clearMode(); } self.notify(); }); } /* constructor() */ get sysDarkPrefers() { return window.matchMedia("(prefers-color-scheme: dark)"); } get isSysDarkPrefer() { return this.sysDarkPrefers.matches; } get isDarkMode() { return this.mode === ModeToggle.DARK_MODE; } get isLightMode() { return this.mode === ModeToggle.LIGHT_MODE; } get hasMode() { return this.mode != null; } get mode() { return sessionStorage.getItem(ModeToggle.MODE_KEY); } /* get the current mode on screen */ get modeStatus() { if (this.isDarkMode || (!this.hasMode && this.isSysDarkPrefer)) { return ModeToggle.DARK_MODE; } else { return ModeToggle.LIGHT_MODE; } } setDark() { $('html').attr(ModeToggle.MODE_ATTR, ModeToggle.DARK_MODE); sessionStorage.setItem(ModeToggle.MODE_KEY, ModeToggle.DARK_MODE); } setLight() { $('html').attr(ModeToggle.MODE_ATTR, ModeToggle.LIGHT_MODE); sessionStorage.setItem(ModeToggle.MODE_KEY, ModeToggle.LIGHT_MODE); } clearMode() { $('html').removeAttr(ModeToggle.MODE_ATTR); sessionStorage.removeItem(ModeToggle.MODE_KEY); } /* Notify another plugins that the theme mode has changed */ notify() { window.postMessage({ direction: ModeToggle.ID, message: this.modeStatus }, "*"); } } /* ModeToggle */ const toggle = new ModeToggle(); function flipMode() { if (toggle.hasMode) { if (toggle.isSysDarkPrefer) { if (toggle.isLightMode) { toggle.clearMode(); } else { toggle.setLight(); } } else { if (toggle.isDarkMode) { toggle.clearMode(); } else { toggle.setDark(); } } } else { if (toggle.isSysDarkPrefer) { toggle.setLight(); } else { toggle.setDark(); } } toggle.notify(); } /* flipMode() */ </script><body data-spy="scroll" data-target="#toc" data-topbar-visible="true"><div id="sidebar" class="d-flex flex-column align-items-end"><div class="profile-wrapper text-center"><div id="avatar"> <a href="/" alt="avatar" class="mx-auto"> </a></div><div class="site-title mt-3"> <a href="/">Hubert Effect</a></div><div class="site-subtitle font-italic">Bioinformatics</div></div><ul class="w-100"><li class="nav-item"> <a href="/" class="nav-link"> <i class="fa-fw fas fa-home ml-xl-3 mr-xl-3 unloaded"></i> <span>HOME</span> </a><li class="nav-item"> <a href="/categories/" class="nav-link"> <i class="fa-fw fas fa-stream ml-xl-3 mr-xl-3 unloaded"></i> <span>CATEGORIES</span> </a><li class="nav-item"> <a href="/tags/" class="nav-link"> <i class="fa-fw fas fa-tag ml-xl-3 mr-xl-3 unloaded"></i> <span>TAGS</span> </a><li class="nav-item"> <a href="/archives/" class="nav-link"> <i class="fa-fw fas fa-archive ml-xl-3 mr-xl-3 unloaded"></i> <span>ARCHIVES</span> </a><li class="nav-item"> <a href="/about/" class="nav-link"> <i class="fa-fw fas fa-info-circle ml-xl-3 mr-xl-3 unloaded"></i> <span>ABOUT</span> </a><li class="nav-item"> <a href="/moment/" class="nav-link"> <i class="fa-fw fas fa-info-circle ml-xl-3 mr-xl-3 unloaded"></i> <span>MOMENT</span> </a></ul><div class="sidebar-bottom mt-auto d-flex flex-wrap justify-content-center align-items-center"> <button class="mode-toggle btn" aria-label="Switch Mode"> <i class="fas fa-adjust"></i> </button> <span class="icon-border"></span> <a href="https://github.com/hubert-bioinformatics" aria-label="github" target="_blank" rel="noopener"> <i class="fab fa-github"></i> </a> <a href="https://twitter.com/twitter_username" aria-label="twitter" target="_blank" rel="noopener"> <i class="fab fa-twitter"></i> </a> <a href=" javascript:location.href = 'mailto:' + ['kjhyug93','gmail.com'].join('@')" aria-label="email" > <i class="fas fa-envelope"></i> </a> <a href="/feed.xml" aria-label="rss" > <i class="fas fa-rss"></i> </a></div></div><div id="topbar-wrapper" class="row justify-content-center"><div id="topbar" class="col-11 d-flex h-100 align-items-center justify-content-between"> <span id="breadcrumb"> <span> <a href="/"> Home </a> </span> <span>Transcriptome I - Introduction to Transcriptome</span> </span> <i id="sidebar-trigger" class="fas fa-bars fa-fw"></i><div id="topbar-title"> Post</div><i id="search-trigger" class="fas fa-search fa-fw"></i> <span id="search-wrapper" class="align-items-center"> <i class="fas fa-search fa-fw"></i> <input class="form-control" id="search-input" type="search" aria-label="search" autocomplete="off" placeholder="Search..."> </span> <span id="search-cancel" >Cancel</span></div></div><div id="main-wrapper"><div id="main"><div class="row"><div id="core-wrapper" class="col-12 col-lg-11 col-xl-8"><div class="post pl-1 pr-1 pl-sm-2 pr-sm-2 pl-md-4 pr-md-4"><h1 data-toc-skip>Transcriptome I - Introduction to Transcriptome</h1><div class="post-meta text-muted"><div> By <em> <a href="https://twitter.com/username">Jonghyuk Kim</a> </em></div><div class="d-flex"><div> <span> Posted <em class="timeago" data-ts="1668034732" data-toggle="tooltip" data-placement="bottom" data-tooltip-df="llll" > 2022-11-10 </em> </span> <span> Updated <em class="timeago" data-ts="1669099527" data-toggle="tooltip" data-placement="bottom" data-tooltip-df="llll" > 2022-11-22 </em> </span> <span class="readtime" data-toggle="tooltip" data-placement="bottom" title="1300 words"> <em>7 min</em> read</span></div></div></div><div class="post-content"><p>본 post는 국가생명연구자원정보센터(KOBIC) 주관 이화여자대학교 생명과학과 이상혁 교수님의 <a href="https://www.edwith.org/transcriptome/lecture/1382678," title="전사체 데이터 분석">전사체 데이터 분석</a>를 정리한 내용입니다.</p><h2 id="intro"><span class="mr-2">Intro</span><a href="#intro" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2></h2><hr /><p>RNA-seq과 microarray에 대한 전반적인 내용을 알아봅니다. <br /><br /></p><h2 id="전사체-연구란"><span class="mr-2">전사체 연구란?</span><a href="#전사체-연구란" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2></h2><hr /><ul><li>Omics: 어떤 대상을 모두 모아서 연구하는 학문<li>Transcriptome 연구: 다양한 상황에서 <strong>RNA 전체를 대상</strong>으로 하는 연구<li>유전자 활동: 유전자 발현의 결과물(RNA, Protein 등)로 판단 <br /><br /></ul><h2 id="유전자-발현"><span class="mr-2">유전자 발현</span><a href="#유전자-발현" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2></h2><hr /><ul><li>where?<ul><li>tissue or organ profile. 장소에 따라 어떤 gene이 발현되느냐가 달라집니다.</ul><li>Under certain circumstances?<ul><li>temporal progress. 동일한 organ 내에서라도 시간에 따라 normal cell이 cancer cell로 변하면서 발현되는 gene의 종류와 양도 달라집니다.</ul><li>Gene expression and Gene regulation<ul><li>a “proxy” measure for transcription/translation events! RNA/protein의 발현 정도로 발현/조절을 파악할 수 있습니다. <br /><br /></ul></ul><h2 id="omics-연구의-장점"><span class="mr-2">Omics 연구의 장점</span><a href="#omics-연구의-장점" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2></h2><hr /><ul><li>survey of all components<ul><li>efficient for “biomarker discovery”<li>unbiased view! “system biology” <br /><br /></ul></ul><h2 id="transcriptome-연구를-통해-해결할-수-있는-문제들"><span class="mr-2">Transcriptome 연구를 통해 해결할 수 있는 문제들</span><a href="#transcriptome-연구를-통해-해결할-수-있는-문제들" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2></h2><hr /><ul><li>compare two (or more) conditions to identify differentially expressed genes<ul><li>control vs. treatment<li>disease vs. normal</ul><li>exploratory analysis<ul><li>what genes are expressed in response to drought stress?<li>what gene expression changes occur during normal retinal development?</ul><li>diagnostic &amp; prognostic tool development<ul><li>can we predict certain conditions (breast cancer vs. normal) 암 진단법을 만들 수 있습니다.<li>can we identify patterns of gene expression that predict a patient’s response to treatment/drug? 치료 예후예측</ul></ul><p>사람의 몸에 침입한 antigen은 antigen-presenting cell에 의하여 우리 몸이 침입을 감지하고 대응할 수 있도록 알리는 역할을 합니다. Cytotoxic T-cell이 직접 antigen을 공격하는 동시에 helper T-cell이 B-cell로 하여금 antibody를 생성, 배출하도록 리드합니다. 또한 memory T-cell로 하여금 적응면역을 갖도록 합니다. <br /><br /></p><h2 id="applications-of-gene-expression"><span class="mr-2">Applications of Gene Expression</span><a href="#applications-of-gene-expression" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2></h2><hr /><ul><li>clustering<li>gene function inference<ul><li>regulatory network</ul><li>molecular pathology<ul><li>classification<li>diagnosis and prognosis <br /><br /></ul></ul><h2 id="transcriptome-연구-범위"><span class="mr-2">Transcriptome 연구 범위</span><a href="#transcriptome-연구-범위" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2></h2><hr /><ul><li>exporession profiling<ul><li>mRNA, miRNA, IncRNA, …<li>alternative splicing</ul><li>differentially expressed genes (DEGs)<li>gene set and pathway analysis<li>precision medicine<ul><li>fusion gene analysis<li>patient stratification<li>cell type and composition inference <br /><br /></ul></ul><h2 id="transcriptome-연구-방법"><span class="mr-2">Transcriptome 연구 방법</span><a href="#transcriptome-연구-방법" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2></h2><hr /><ul><li>microarray: universal biochemistry platforms<br /> <img data-src="/assets/img/post/transcriptome-fig1.png" alt="Post-Image" data-proofer-ignore> <em>microarray<br /> https://www.edwith.org/transcriptome/lecture/1382678</em><br /><ul><li>micro(작은) + array(행렬): 2d 배열의 작은 hole에 물질을 넣고 분석할 수 있는 방식<li>motivations: hundreds, thousands or even millions of individual experiments are conducted <strong>in parallel</strong>, with very few reagents. 즉, 동시에 20,000개~30,000개의 gene의 발현을 한 번에 알 수 있다는 점에서 효율적입니다.<li>many different types<ul><li>DNA<li>mRNA in cDNA<li>proteins (antbodies)<li>chemical compounds<li>tissues<li>carbohydrates</ul><li>principles<ul><li>extension of southern and northern blotting<li>microarray는 gene에 specific한 sequence로 구성된 fixed probes가 hole마다 고정되어 있습니다. (hybridization-bases)<li>normal vs. tumor의 mRNA를 각각 green, red dye를 붙인 뒤 1:1 비율로 microarray에 loading하면 normal vs. tumor에서 더 많이 발현되는 색상이 우세하게 나타납니다.<br /> <img data-src="/assets/img/post/transcriptome-fig2.png" alt="Post-Image" data-proofer-ignore> <em>microarray principles<br /> https://www.edwith.org/transcriptome/lecture/1382678</em><br /></ul><li>Affymetrix expression arrays<ul><li>Affymetrix사에서 만든 expression arrays로, 25-mers 길이의 oligo로 구성되었습니다.<li>Human Transcriptome Array 2.0은 285,000개 이상의 RNA 서열을 대상으로 발현양을 확인할 수 있습니다.<br /> <img data-src="/assets/img/post/transcriptome-fig3.png" alt="Post-Image" data-proofer-ignore> <em>Affymetrix Expression Arrays<br /> https://www.edwith.org/transcriptome/lecture/1382678</em><br /></ul></ul><li>RNA-Seq (Transcriptome Sequencing)<ul><li>the high throughput sequencing of cDNA using NGS technology<li>the goal is to identify regions in the genome that are being transcribed in a sample<li>strategies for reconstructing transcripts from RNA-Seq<ul><li>align-then-assemble: human reference genome에 align 한 뒤 gene별로 얼마나 붙는지 확인합니다. Quantification<li>assemble-then-align: sequencing reads끼리 assembly 한 뒤 reference genome에 align 합니다. Novel RNAs를 확인할 수 있습니다.<br /> <img data-src="/assets/img/post/transcriptome-fig4.png" alt="Post-Image" data-proofer-ignore> <em>RNA-Seq<br /> https://www.edwith.org/transcriptome/lecture/1382678</em><br /></ul><li>common analysis of RNA-Seq<ul><li>gene expression analysis<li>differential expression analysis<li>transcript discovery and annotation<ul><li>novel transcripts/genes<li>noncoding RNAs: miRNAs, IncRNAs, …</ul><li>gene fusion detection (structural variants)<li>alternative splicing event<li>RNA-specific phenomena<ul><li>allele-specific expression: 부/모 어느 쪽에서 온 allele이 발현되는지 확인합니다.<li>RNA editing</ul><li>Mutation discovery &amp; prioritization</ul><li>advantages of RNA-Seq over microarray<ul><li>one sample reparation, various analysis<li>RNA-Seq has higher resolution<ul><li>single-base pair resolution<li>dynamic range of gene expression</ul><li>variatin of RNA-Seq<ul><li>single cell sequencing<li>miRNA sequencing<li>Long read sequencing</ul></ul><li>challenges of RNA-Seq<ul><li>RNA is fragile compared to DNA (easily degraded) 샘플 변성이 매우 잘 일어납니다.<li>Sample!!! purifu? quantity? quality?<li>RNAs consist of small exons separated by introns<li>the relative abundance of RNAs vary wildly<li>RNAs come in a wide range of sizes</ul><li>RNA-Seq library enrichment strategies<ul><li>total RNA: all types RNAs가 sequencing 됩니다.<li>rRNA reduction: rRNAs를 제거한 나머지 RNAs만 sequencing 합니다.<li>PolyA selection: mRNA만 선별적으로 sequencing 합니다.<li>cDNA capture: interesting genes의 mRNA만 선택하여 sequencing 합니다.<br /> <img data-src="/assets/img/post/transcriptome-fig5.png" alt="Post-Image" data-proofer-ignore> <em>RNA-Seq library enrichment strategies<br /> https://www.edwith.org/transcriptome/lecture/1382678</em><br /></ul><li>stranded vs. unstranded<ul><li>최근에는 대부분 stranded sequencing을 진행합니다.<br /> <img data-src="/assets/img/post/transcriptome-fig6.png" alt="Post-Image" data-proofer-ignore> <em>stranded vs. unstranded<br /> https://www.edwith.org/transcriptome/lecture/1382678</em><br /></ul><li>RNA quality control<ul><li>RIN: RNA integrity number. 0(bad) ~ 10(good). RNA quality를 나타내는 수치입니다.<br /> <img data-src="/assets/img/post/transcriptome-fig7.png" alt="Post-Image" data-proofer-ignore> <em>RIN<br /> https://www.edwith.org/transcriptome/lecture/1382678</em><br /></ul><li>replicates<ul><li>biological replicate: multiple isolations of cells showing <strong>the same phenotype, stage</strong> of other experimental condition. 즉, 시료의 양이 충분하다면 가급적 3개 이상으로 시료를 나누어 sequencing, analysis 하는 것을 권장합니다. 3개 중 1개 데이터에 이상이 있더라도 나머지 2개 데이터를 통계처리하여 올바른 결과를 도출할 수 있기 때문입니다. <br /><br /></ul></ul></ul><h2 id="take-home-message"><span class="mr-2">Take Home Message</span><a href="#take-home-message" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2></h2><hr /><p>microarray 대비 RNA-Seq이 가지는 장점을 확인할 수 있었습니다. RNA-Seq에서 샘플의 quality가 매우 중요하고, 얻은 data로부터 gene expression level, gene fusion analysis, mutation discovery 등 다양한 결과를 얻을 수 있음을 확인 했습니다.</p></div><div class="post-tail-wrapper text-muted"><div class="post-meta mb-3"> <i class="far fa-folder-open fa-fw mr-1"></i> <a href='/categories/bioinformatics/'>Bioinformatics</a>, <a href='/categories/ngs/'>NGS</a></div><div class="post-tags"> <i class="fa fa-tags fa-fw mr-1"></i> <a href="/tags/bi/" class="post-tag no-text-decoration" >BI</a> <a href="/tags/bioinformatics/" class="post-tag no-text-decoration" >bioinformatics</a> <a href="/tags/transcriptome/" class="post-tag no-text-decoration" >transcriptome</a></div><div class="post-tail-bottom d-flex justify-content-between align-items-center mt-3 pt-5 pb-2"><div class="license-wrapper"> This post is licensed under <a href="https://creativecommons.org/licenses/by/4.0/"> CC BY 4.0 </a> by the author.</div><div class="share-wrapper"> <span class="share-label text-muted mr-1">Share</span> <span class="share-icons"> <a href="https://twitter.com/intent/tweet?text=Transcriptome I - Introduction to Transcriptome - Hubert Effect&amp;url=/posts/Transcriptome1/" data-toggle="tooltip" data-placement="top" title="Twitter" target="_blank" rel="noopener" aria-label="Twitter"> <i class="fa-fw fab fa-twitter"></i> </a> <a href="https://www.facebook.com/sharer/sharer.php?title=Transcriptome I - Introduction to Transcriptome - Hubert Effect&amp;u=/posts/Transcriptome1/" data-toggle="tooltip" data-placement="top" title="Facebook" target="_blank" rel="noopener" aria-label="Facebook"> <i class="fa-fw fab fa-facebook-square"></i> </a> <a href="https://t.me/share/url?url=/posts/Transcriptome1/&amp;text=Transcriptome I - Introduction to Transcriptome - Hubert Effect" data-toggle="tooltip" data-placement="top" title="Telegram" target="_blank" rel="noopener" aria-label="Telegram"> <i class="fa-fw fab fa-telegram"></i> </a> <i id="copy-link" class="fa-fw fas fa-link small" data-toggle="tooltip" data-placement="top" title="Copy link" data-title-succeed="Link copied successfully!"> </i> </span></div></div></div></div></div><div id="panel-wrapper" class="col-xl-3 pl-2 text-muted"><div class="access"><div id="access-lastmod" class="post"><div class="panel-heading">Recently Updated</div><ul class="post-content pl-0 pb-1 ml-1 mt-2"><li><a href="/posts/Certificate_predict_protein_structure_using_alphafold/">(LAIDD) 알파폴드를 이용한 단백질 구조 예측 및 평가</a><li><a href="/posts/Epigenetics1/">Epigenetics I - 후성유전학의 개요</a><li><a href="/posts/Epigenetics6/">Epigenetics VI - 후성유전체 데이터 생산과 분석1</a><li><a href="/posts/AMP_guideline/">AMP Guideline</a><li><a href="/posts/R_for_Bioinformatics_4/">생물정보학을 위한 R프로그래밍 IV - 데이터 취득과 정제</a></ul></div><div id="access-tags"><div class="panel-heading">Trending Tags</div><div class="d-flex flex-wrap mt-3 mb-1 mr-3"> <a class="post-tag" href="/tags/certificate/">certificate</a> <a class="post-tag" href="/tags/bioinformatics/">bioinformatics</a> <a class="post-tag" href="/tags/bi/">BI</a> <a class="post-tag" href="/tags/python/">python</a> <a class="post-tag" href="/tags/laidd/">LAIDD</a> <a class="post-tag" href="/tags/ngs/">NGS</a> <a class="post-tag" href="/tags/statistics/">statistics</a> <a class="post-tag" href="/tags/kobic/">KOBIC</a> <a class="post-tag" href="/tags/biology/">biology</a> <a class="post-tag" href="/tags/study/">study</a></div></div></div><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@1.0.1/dist/bootstrap-toc.min.js"></script><div id="toc-wrapper" class="pl-0 pr-4 mb-5"><div class="panel-heading pl-3 pt-2 mb-2">Contents</div><nav id="toc" data-toggle="toc"></nav></div></div></div><div class="row"><div class="col-12 col-lg-11 col-xl-8"><div id="tail-wrapper" class="pl-1 pr-1 pl-sm-2 pr-sm-2 pl-md-4 pr-md-4"><div id="related-posts" class="mt-5 mb-2 mb-sm-4"><h3 class="pt-2 mt-1 mb-4 ml-1" data-toc-skip>Further Reading</h3><div class="card-deck mb-4"><div class="card"> <a href="/posts/Transcriptome2/"><div class="card-body"> <em class="timeago small" data-ts="1669070832" > 2022-11-22 </em><h3 class="pt-0 mt-1 mb-3" data-toc-skip>Transcriptome II - Data Preprocessing</h3><div class="text-muted small"><p> 본 post는 국가생명연구자원정보센터(KOBIC) 주관 이화여자대학교 생명과학과 이상혁 교수님의 전사체 데이터 분석를 정리한 내용입니다. Intro RNA-seq data의 mapping, normalization, quantification에 대해 알아봅니다. Quality Control Sequencing data를 얻으면 base ...</p></div></div></a></div><div class="card"> <a href="/posts/Transcriptome3/"><div class="card-body"> <em class="timeago small" data-ts="1669087012" > 2022-11-22 </em><h3 class="pt-0 mt-1 mb-3" data-toc-skip>Transcriptome III - Differential Expression & Visual Exploration</h3><div class="text-muted small"><p> 본 post는 국가생명연구자원정보센터(KOBIC) 주관 이화여자대학교 생명과학과 이상혁 교수님의 전사체 데이터 분석를 정리한 내용입니다. Intro RNA-seq data의 differential exporession, visual exploration에 대해 알아봅니다. Differential Expression Typical Proces...</p></div></div></a></div><div class="card"> <a href="/posts/Longread_sequencing1/"><div class="card-body"> <em class="timeago small" data-ts="1688595813" > 2023-07-06 </em><h3 class="pt-0 mt-1 mb-3" data-toc-skip>Long Read Sequencing I - Introduction</h3><div class="text-muted small"><p> 본 post는 국가생명연구자원정보센터(KOBIC) 주관 서울대학교 박사 후 연구원 김준님의 롱리드 시퀀싱 개요를 정리한 내용입니다. Intro 시권싱 기법과 이를 활용한 생물학 연구 방법론에 대해 알아봅니다. 롱리드 시퀀싱과 기존 숏리드 시퀀싱 기법의 차이를 알아봅니다. 롱리드 시퀀싱 기법을 활용...</p></div></div></a></div></div></div><div class="post-navigation d-flex justify-content-between"> <a href="/posts/Certificate_cancer_genomics_analysis/" class="btn btn-outline-primary" prompt="Older"><p>(EDWITH-KOBIC) 암 유전체 분석</p></a> <a href="/posts/database-sql_3/" class="btn btn-outline-primary" prompt="Newer"><p>SQL 기초</p></a></div></div></div></div><footer class="d-flex w-100 justify-content-center"><div class="d-flex justify-content-between align-items-center text-muted"><div class="footer-left"><p class="mb-0"> © 2025 <a href="https://twitter.com/username">Jonghyuk Kim</a>. <span data-toggle="tooltip" data-placement="top" title="Except where otherwise noted, the blog posts on this site are licensed under the Creative Commons Attribution 4.0 International (CC BY 4.0) License by the author.">Some rights reserved.</span></p></div><div class="footer-right"><p class="mb-0"> Powered by <a href="https://jekyllrb.com" target="_blank" rel="noopener">Jekyll</a> with <a href="https://github.com/cotes2020/jekyll-theme-chirpy" target="_blank" rel="noopener">Chirpy</a> theme.</p></div></div></footer></div><div id="search-result-wrapper" class="d-flex justify-content-center unloaded"><div class="col-12 col-sm-11 post-content"><div id="search-hints"><div id="access-tags"><div class="panel-heading">Trending Tags</div><div class="d-flex flex-wrap mt-3 mb-1 mr-3"> <a class="post-tag" href="/tags/certificate/">certificate</a> <a class="post-tag" href="/tags/bioinformatics/">bioinformatics</a> <a class="post-tag" href="/tags/bi/">BI</a> <a class="post-tag" href="/tags/python/">python</a> <a class="post-tag" href="/tags/laidd/">LAIDD</a> <a class="post-tag" href="/tags/ngs/">NGS</a> <a class="post-tag" href="/tags/statistics/">statistics</a> <a class="post-tag" href="/tags/kobic/">KOBIC</a> <a class="post-tag" href="/tags/biology/">biology</a> <a class="post-tag" href="/tags/study/">study</a></div></div></div><div id="search-results" class="d-flex flex-wrap justify-content-center text-muted mt-3"></div></div></div></div><div id="mask"></div><a id="back-to-top" href="#" aria-label="back-to-top" class="btn btn-lg btn-box-shadow" role="button"> <i class="fas fa-angle-up"></i> </a> <script src="https://cdn.jsdelivr.net/npm/simple-jekyll-search@1.10.0/dest/simple-jekyll-search.min.js"></script> <script> SimpleJekyllSearch({ searchInput: document.getElementById('search-input'), resultsContainer: document.getElementById('search-results'), json: '/assets/js/data/search.json', searchResultTemplate: '<div class="pl-1 pr-1 pl-sm-2 pr-sm-2 pl-lg-4 pr-lg-4 pl-xl-0 pr-xl-0"> <a href="{url}">{title}</a><div class="post-meta d-flex flex-column flex-sm-row text-muted mt-1 mb-1"> {categories} {tags}</div><p>{snippet}</p></div>', noResultsText: '<p class="mt-5">Oops! No result founds.</p>', templateMiddleware: function(prop, value, template) { if (prop === 'categories') { if (value === '') { return `${value}`; } else { return `<div class="mr-sm-4"><i class="far fa-folder fa-fw"></i>${value}</div>`; } } if (prop === 'tags') { if (value === '') { return `${value}`; } else { return `<div><i class="fa fa-tag fa-fw"></i>${value}</div>`; } } } }); </script> <script src="https://cdn.jsdelivr.net/combine/npm/magnific-popup@1/dist/jquery.magnific-popup.min.js,npm/lozad/dist/lozad.min.js,npm/clipboard@2/dist/clipboard.min.js"></script> <script src="https://cdn.jsdelivr.net/combine/npm/dayjs@1/dayjs.min.js,npm/dayjs@1/locale/en.min.js,npm/dayjs@1/plugin/relativeTime.min.js,npm/dayjs@1/plugin/localizedFormat.min.js"></script> <script defer src="/assets/js/dist/post.min.js"></script> <script> /* see: <https://docs.mathjax.org/en/latest/options/input/tex.html#tex-options> */ MathJax = { tex: { inlineMath: [ /* start/end delimiter pairs for in-line math */ ['$','$'], ['\\(','\\)'] ], displayMath: [ /* start/end delimiter pairs for display math */ ['$$', '$$'], ['\\[', '\\]'] ] } }; </script> <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js"> </script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4/dist/js/bootstrap.bundle.min.js"></script> <script defer src="/app.js"></script>
