<!DOCTYPE html><html lang="en" ><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><meta name="generator" content="Jekyll v4.2.2" /><meta property="og:title" content="Experimental Design for Single Cell Analysis" /><meta property="og:locale" content="en" /><meta name="description" content="본 post는 국가생명연구자원정보센터(KOBIC) 주관 광주과학기술원(GIST) 박지환 교수님의 단일세포 분석 소개 및 실험 디자인을 정리한 내용입니다." /><meta property="og:description" content="본 post는 국가생명연구자원정보센터(KOBIC) 주관 광주과학기술원(GIST) 박지환 교수님의 단일세포 분석 소개 및 실험 디자인을 정리한 내용입니다." /><link rel="canonical" href="/posts/Experimental_Design_for_Single_Cell_Analysis/" /><meta property="og:url" content="/posts/Experimental_Design_for_Single_Cell_Analysis/" /><meta property="og:site_name" content="Hubert Effect" /><meta property="og:type" content="article" /><meta property="article:published_time" content="2022-10-03T14:26:55+09:00" /><meta name="twitter:card" content="summary" /><meta property="twitter:title" content="Experimental Design for Single Cell Analysis" /><meta name="twitter:site" content="@twitter_username" /> <script type="application/ld+json"> {"@context":"https://schema.org","@type":"BlogPosting","dateModified":"2022-10-03T14:26:55+09:00","datePublished":"2022-10-03T14:26:55+09:00","description":"본 post는 국가생명연구자원정보센터(KOBIC) 주관 광주과학기술원(GIST) 박지환 교수님의 단일세포 분석 소개 및 실험 디자인을 정리한 내용입니다.","headline":"Experimental Design for Single Cell Analysis","mainEntityOfPage":{"@type":"WebPage","@id":"/posts/Experimental_Design_for_Single_Cell_Analysis/"},"url":"/posts/Experimental_Design_for_Single_Cell_Analysis/"}</script><title>Experimental Design for Single Cell Analysis | Hubert Effect</title><link rel="apple-touch-icon" sizes="180x180" href="/assets/img/favicons/apple-touch-icon.png"><link rel="icon" type="image/png" sizes="32x32" href="/assets/img/favicons/favicon-32x32.png"><link rel="icon" type="image/png" sizes="16x16" href="/assets/img/favicons/favicon-16x16.png"><link rel="manifest" href="/assets/img/favicons/site.webmanifest"><link rel="shortcut icon" href="/assets/img/favicons/favicon.ico"><meta name="apple-mobile-web-app-title" content="Hubert Effect"><meta name="application-name" content="Hubert Effect"><meta name="msapplication-TileColor" content="#da532c"><meta name="msapplication-config" content="/assets/img/favicons/browserconfig.xml"><meta name="theme-color" content="#ffffff"><link rel="preconnect" href="https://fonts.googleapis.com" ><link rel="dns-prefetch" href="https://fonts.googleapis.com" ><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link rel="dns-prefetch" href="https://fonts.gstatic.com" crossorigin><link rel="preconnect" href="https://fonts.googleapis.com" ><link rel="dns-prefetch" href="https://fonts.googleapis.com" ><link rel="preconnect" href="https://cdn.jsdelivr.net" ><link rel="dns-prefetch" href="https://cdn.jsdelivr.net" ><link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Lato&family=Source+Sans+Pro:wght@400;600;700;900&display=swap"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4/dist/css/bootstrap.min.css"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.11.2/css/all.min.css"><link rel="stylesheet" href="/assets/css/style.css"><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@1.0.1/dist/bootstrap-toc.min.css"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/magnific-popup@1/dist/magnific-popup.min.css"> <script src="https://cdn.jsdelivr.net/npm/jquery@3/dist/jquery.min.js"></script> <script type="text/javascript"> class ModeToggle { static get MODE_KEY() { return "mode"; } static get MODE_ATTR() { return "data-mode"; } static get DARK_MODE() { return "dark"; } static get LIGHT_MODE() { return "light"; } static get ID() { return "mode-toggle"; } constructor() { if (this.hasMode) { if (this.isDarkMode) { if (!this.isSysDarkPrefer) { this.setDark(); } } else { if (this.isSysDarkPrefer) { this.setLight(); } } } let self = this; /* always follow the system prefers */ this.sysDarkPrefers.addEventListener("change", () => { if (self.hasMode) { if (self.isDarkMode) { if (!self.isSysDarkPrefer) { self.setDark(); } } else { if (self.isSysDarkPrefer) { self.setLight(); } } self.clearMode(); } self.notify(); }); } /* constructor() */ get sysDarkPrefers() { return window.matchMedia("(prefers-color-scheme: dark)"); } get isSysDarkPrefer() { return this.sysDarkPrefers.matches; } get isDarkMode() { return this.mode === ModeToggle.DARK_MODE; } get isLightMode() { return this.mode === ModeToggle.LIGHT_MODE; } get hasMode() { return this.mode != null; } get mode() { return sessionStorage.getItem(ModeToggle.MODE_KEY); } /* get the current mode on screen */ get modeStatus() { if (this.isDarkMode || (!this.hasMode && this.isSysDarkPrefer)) { return ModeToggle.DARK_MODE; } else { return ModeToggle.LIGHT_MODE; } } setDark() { $('html').attr(ModeToggle.MODE_ATTR, ModeToggle.DARK_MODE); sessionStorage.setItem(ModeToggle.MODE_KEY, ModeToggle.DARK_MODE); } setLight() { $('html').attr(ModeToggle.MODE_ATTR, ModeToggle.LIGHT_MODE); sessionStorage.setItem(ModeToggle.MODE_KEY, ModeToggle.LIGHT_MODE); } clearMode() { $('html').removeAttr(ModeToggle.MODE_ATTR); sessionStorage.removeItem(ModeToggle.MODE_KEY); } /* Notify another plugins that the theme mode has changed */ notify() { window.postMessage({ direction: ModeToggle.ID, message: this.modeStatus }, "*"); } } /* ModeToggle */ const toggle = new ModeToggle(); function flipMode() { if (toggle.hasMode) { if (toggle.isSysDarkPrefer) { if (toggle.isLightMode) { toggle.clearMode(); } else { toggle.setLight(); } } else { if (toggle.isDarkMode) { toggle.clearMode(); } else { toggle.setDark(); } } } else { if (toggle.isSysDarkPrefer) { toggle.setLight(); } else { toggle.setDark(); } } toggle.notify(); } /* flipMode() */ </script><body data-spy="scroll" data-target="#toc" data-topbar-visible="true"><div id="sidebar" class="d-flex flex-column align-items-end"><div class="profile-wrapper text-center"><div id="avatar"> <a href="/" alt="avatar" class="mx-auto"> </a></div><div class="site-title mt-3"> <a href="/">Hubert Effect</a></div><div class="site-subtitle font-italic">Bioinformatics</div></div><ul class="w-100"><li class="nav-item"> <a href="/" class="nav-link"> <i class="fa-fw fas fa-home ml-xl-3 mr-xl-3 unloaded"></i> <span>HOME</span> </a><li class="nav-item"> <a href="/categories/" class="nav-link"> <i class="fa-fw fas fa-stream ml-xl-3 mr-xl-3 unloaded"></i> <span>CATEGORIES</span> </a><li class="nav-item"> <a href="/tags/" class="nav-link"> <i class="fa-fw fas fa-tag ml-xl-3 mr-xl-3 unloaded"></i> <span>TAGS</span> </a><li class="nav-item"> <a href="/archives/" class="nav-link"> <i class="fa-fw fas fa-archive ml-xl-3 mr-xl-3 unloaded"></i> <span>ARCHIVES</span> </a><li class="nav-item"> <a href="/about/" class="nav-link"> <i class="fa-fw fas fa-info-circle ml-xl-3 mr-xl-3 unloaded"></i> <span>ABOUT</span> </a><li class="nav-item"> <a href="/moment/" class="nav-link"> <i class="fa-fw fas fa-info-circle ml-xl-3 mr-xl-3 unloaded"></i> <span>MOMENT</span> </a></ul><div class="sidebar-bottom mt-auto d-flex flex-wrap justify-content-center align-items-center"> <button class="mode-toggle btn" aria-label="Switch Mode"> <i class="fas fa-adjust"></i> </button> <span class="icon-border"></span> <a href="https://github.com/hubert-bioinformatics" aria-label="github" target="_blank" rel="noopener"> <i class="fab fa-github"></i> </a> <a href="https://twitter.com/twitter_username" aria-label="twitter" target="_blank" rel="noopener"> <i class="fab fa-twitter"></i> </a> <a href=" javascript:location.href = 'mailto:' + ['kjhyug93','gmail.com'].join('@')" aria-label="email" > <i class="fas fa-envelope"></i> </a> <a href="/feed.xml" aria-label="rss" > <i class="fas fa-rss"></i> </a></div></div><div id="topbar-wrapper" class="row justify-content-center"><div id="topbar" class="col-11 d-flex h-100 align-items-center justify-content-between"> <span id="breadcrumb"> <span> <a href="/"> Home </a> </span> <span>Experimental Design for Single Cell Analysis</span> </span> <i id="sidebar-trigger" class="fas fa-bars fa-fw"></i><div id="topbar-title"> Post</div><i id="search-trigger" class="fas fa-search fa-fw"></i> <span id="search-wrapper" class="align-items-center"> <i class="fas fa-search fa-fw"></i> <input class="form-control" id="search-input" type="search" aria-label="search" autocomplete="off" placeholder="Search..."> </span> <span id="search-cancel" >Cancel</span></div></div><div id="main-wrapper"><div id="main"><div class="row"><div id="core-wrapper" class="col-12 col-lg-11 col-xl-8"><div class="post pl-1 pr-1 pl-sm-2 pr-sm-2 pl-md-4 pr-md-4"><h1 data-toc-skip>Experimental Design for Single Cell Analysis</h1><div class="post-meta text-muted"><div> By <em> <a href="https://twitter.com/username">Jonghyuk Kim</a> </em></div><div class="d-flex"><div> <span> Posted <em class="timeago" data-ts="1664774815" data-toggle="tooltip" data-placement="bottom" data-tooltip-df="llll" > 2022-10-03 </em> </span> <span class="readtime" data-toggle="tooltip" data-placement="bottom" title="1278 words"> <em>7 min</em> read</span></div></div></div><div class="post-content"><p>본 post는 국가생명연구자원정보센터(KOBIC) 주관 광주과학기술원(GIST) 박지환 교수님의 <a href="https://www.edwith.org/single-cell/joinLectures/361836," title="단일세포 분석 소개 및 실험 디자인">단일세포 분석 소개 및 실험 디자인</a>을 정리한 내용입니다.</p><h2 id="intro"><span class="mr-2">Intro</span><a href="#intro" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2></h2><hr /><p>Single cell analysis의 필요성 및 연구배경, 실험 디자인에 대한 내용을 이해합니다. <br /><br /></p><p><img data-src="/assets/img/post/SC-intro.png" alt="Post-Image" data-proofer-ignore> <em>Single Cell Analysis 필요성<br /> https://www.edwith.org/single-cell/joinLectures/361836</em> <br /><br /></p><p>과거에는 RT-PCR, bulk RNA-seq 등으로 cell에서 gene expression level을 확인했습니다. 그 결과 structure, location, function, a small number of markers 등에 따라 사람은 약 200여 개 종류의 서로 다른 cell types이 존재한다고 알려졌습니다. 하지만 single cell analysis를 통해서 동일한 organ이라도 서로 다른 cell type이 존재함을 확인했습니다. 또한 한 사람당 약 20,000개의 cell과 그 곳에 포함된 약 20,000~30,000개의 gene에 대해 분석하고 더 정확한 결과를 얻을 수 있게 되었습니다. <br /><br /></p><h2 id="single-cell-analysis-필요성"><span class="mr-2">Single Cell Analysis 필요성</span><a href="#single-cell-analysis-필요성" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2></h2><hr /><p><img data-src="/assets/img/post/SC-purpose1.png" alt="Post-Image" data-proofer-ignore> <em>Single Cell Analysis 필요성<br /> https://www.edwith.org/single-cell/joinLectures/361836</em> <br /><br /></p><p>RNA-seq은 cell을 구분하지 못하고 평균적인 gene expression level을 확인합니다.</p><p>반면 single cell analysis는 cell별로 gene exporession level을 구분하여 확인할 수 있습니다. <br /><br /></p><p><img data-src="/assets/img/post/SC-purpose2.png" alt="Post-Image" data-proofer-ignore> <em>Single Cell Analysis 필요성<br /> https://www.edwith.org/single-cell/joinLectures/361836</em> <br /><br /></p><p>또한 single cell analysis를 활용하면 cell specific change와 cell type composition change를 구분할 수 있습니다. 예를 들어 정상인과 환자의 single cell analysis를 진행하면 disease와 연관된 red cell의 green gene expression high level이 cell 내 exporession 증가에 의해서인지(cell specific change), 혹은 red cell의 증가에 의해서인지(cell type coposition change) 구분할 수 있습니다. <br /><br /></p><p><img data-src="/assets/img/post/SC-purpose3.png" alt="Post-Image" data-proofer-ignore> <em>Single Cell Analysis 필요성<br /> https://www.edwith.org/single-cell/joinLectures/361836</em> <br /><br /></p><p>뿐만 아니라 cell이 분화하면서 변화하는 gene expression level을 추적 관찰할 수도 있습니다. 더 나아가 생물의 진화 양상을 추적할 수도 있습니다. <br /><br /></p><h2 id="basic-steps-of-single-cell-analysis"><span class="mr-2">Basic Steps of Single Cell Analysis</span><a href="#basic-steps-of-single-cell-analysis" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2></h2><hr /><p><img data-src="/assets/img/post/SC-step1.png" alt="Post-Image" data-proofer-ignore> <em>Single Cell Analysis 기본 단계<br /> https://www.edwith.org/single-cell/joinLectures/361836</em> <br /><br /></p><p>Single cell analysis의 기본적인 단계입니다. Tissue에서 cell을 분리한 다음 library를 만들고 rna seq을 진행합니다. Gene expression data를 clustering한 뒤 발현양을 비교합니다. <br /><br /></p><p><img data-src="/assets/img/post/SC-step2.png" alt="Post-Image" data-proofer-ignore> <em>Single Cell Analysis: Library Preparation<br /> https://www.edwith.org/single-cell/joinLectures/361836</em> <br /><br /></p><p>Library prep에 사용할 수 있는 방법은 여러 가지 개발되어 있으며 상요화 된 기술도 존재합니다. UMI 사용여부, cDNA를 얼만큼 cover하는지, 그리고 한 번에 몇 개의 cell을 cover할 수 있는지 등에 차이가 있습니다. <br /><br /></p><h2 id="experimental-design-for-single-cell-analysis"><span class="mr-2">Experimental Design for Single Cell Analysis</span><a href="#experimental-design-for-single-cell-analysis" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2></h2><hr /><p>기본적으로 모든 tissue에 대해 single cell analysis를 진행할 수 있습니다. 하지만 tissue 종류에 따라 dissociation 난이도가 다르고 포함하고 있는 total RNA 양도 다르기 때문에, 특성을 고려한 실험 디자인이 필요합니다. <br /><br /></p><p><img data-src="/assets/img/post/SC-step3.png" alt="Post-Image" data-proofer-ignore> <em>Sample Preparation Protocol Varies by Cell-types<br /> https://www.edwith.org/single-cell/joinLectures/361836</em> <br /><br /></p><p>RNA 양이 적은 solid tissue나 adherent cell culture는 dissociation을 수행합니다. 관심있는 cell만 골라낼 필요가 있는 경우에는 MACS나 FAC sorting으로 enrichment를 수행합니다. <br /><br /></p><p><img data-src="/assets/img/post/SC-step4.png" alt="Post-Image" data-proofer-ignore> <em>Doublets Problems<br /> https://www.edwith.org/single-cell/joinLectures/361836</em> <br /><br /></p><p>Single cell analysis의 문제점 중 하나는 바로 doublets입니다. Droplet 방식을 사용하는 single cell anlaysis는 필연적으로 doublelets 문제를 나타내는 것으로 알려져 있는데, cell loading 개수에 따라 그 비율이 증가한다고 알려져 있습니다. <br /><br /></p><p><img data-src="/assets/img/post/SC-step5.png" alt="Post-Image" data-proofer-ignore> <em>Sequencing Depth for scRNAseq<br /> https://www.edwith.org/single-cell/joinLectures/361836</em> <br /><br /></p><p>Single call analysis에 적당한 sequencing depth를 찾기 위한 data입니다. 약 30,000~50,000x에서 적당한 genes 개수와 umi 수를 확인할 수 있습니다. 물론 적당한 depth 또한 tissue types에 따라 달라질 수 있음을 고려해야 합니다.</p><p>또한 흥미로운 사실은 86,503x data에서 확인된 네 개의 cell clustering 결과가 25,000x, 5,000x, 500x로 줄였을 때 변함없이 네 개의 cluster가 나타남을 봤을 때 depth가 cell clustering에는 별다른 영향을 주지 않음을 확인할 수 있습니다. <br /><br /></p><h2 id="chanllenges-and-future-direction-for-scrnaseq"><span class="mr-2">Chanllenges and Future Direction for scRNAseq</span><a href="#chanllenges-and-future-direction-for-scrnaseq" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2></h2><hr /><ol><li><p>The sparsity of data</p><p>Transcript가 적으므로 data signal보다는 noise가 더 많이 나타나는 문제가 있습니다. 따라서 발현량이 0인 gene을 다수 나타납니다. 이는 RNA capture 효율과 cDNA conversion의 문제로 나타난 결과입니다.</p><li><p>Batch effect</p><p>실험자, 어떤 연구실에서 실험했는지, 혹은 어떤 실험조건으로 진행했는지 등에 따라서 batch effect가 발생할 수 있습니다. 이는 computational method나 서로 다른 샘플을 multiplexing하여 해결할 수 있습니다. <br /><br /></p></ol><p><img data-src="/assets/img/post/SC-step6.png" alt="Post-Image" data-proofer-ignore> <em>Integration of Single Cell Multi-omics Data<br /> https://www.edwith.org/single-cell/joinLectures/361836</em> <br /><br /></p><p>미래에는 single cell isolation 이후 genomics부터 metabolomics까지 다양한 multi-omics data를 분석하고 활용할 수 있을 것입니다. <br /><br /></p><h2 id="take-home-message"><span class="mr-2">Take Home Message</span><a href="#take-home-message" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2></h2><hr /><p>Single cell analysis의 원리와 활용 가능성에 대해 알아볼 수 있었습니다. 현재 업무에서 사용중인 genomics data와 disease, treatment와 연관된 database들이 scRNAseq 결과와 결합한다면 personalized medicine에 활용될 수 있음을 엿볼 수 있었습니다.</p></div><div class="post-tail-wrapper text-muted"><div class="post-meta mb-3"> <i class="far fa-folder-open fa-fw mr-1"></i> <a href='/categories/bioinformatics/'>Bioinformatics</a>, <a href='/categories/ngs/'>NGS</a></div><div class="post-tags"> <i class="fa fa-tags fa-fw mr-1"></i> <a href="/tags/bi/" class="post-tag no-text-decoration" >BI</a> <a href="/tags/bioinformatics/" class="post-tag no-text-decoration" >bioinformatics</a> <a href="/tags/single-cell-analysis/" class="post-tag no-text-decoration" >single cell analysis</a></div><div class="post-tail-bottom d-flex justify-content-between align-items-center mt-3 pt-5 pb-2"><div class="license-wrapper"> This post is licensed under <a href="https://creativecommons.org/licenses/by/4.0/"> CC BY 4.0 </a> by the author.</div><div class="share-wrapper"> <span class="share-label text-muted mr-1">Share</span> <span class="share-icons"> <a href="https://twitter.com/intent/tweet?text=Experimental Design for Single Cell Analysis - Hubert Effect&amp;url=/posts/Experimental_Design_for_Single_Cell_Analysis/" data-toggle="tooltip" data-placement="top" title="Twitter" target="_blank" rel="noopener" aria-label="Twitter"> <i class="fa-fw fab fa-twitter"></i> </a> <a href="https://www.facebook.com/sharer/sharer.php?title=Experimental Design for Single Cell Analysis - Hubert Effect&amp;u=/posts/Experimental_Design_for_Single_Cell_Analysis/" data-toggle="tooltip" data-placement="top" title="Facebook" target="_blank" rel="noopener" aria-label="Facebook"> <i class="fa-fw fab fa-facebook-square"></i> </a> <a href="https://t.me/share/url?url=/posts/Experimental_Design_for_Single_Cell_Analysis/&amp;text=Experimental Design for Single Cell Analysis - Hubert Effect" data-toggle="tooltip" data-placement="top" title="Telegram" target="_blank" rel="noopener" aria-label="Telegram"> <i class="fa-fw fab fa-telegram"></i> </a> <i id="copy-link" class="fa-fw fas fa-link small" data-toggle="tooltip" data-placement="top" title="Copy link" data-title-succeed="Link copied successfully!"> </i> </span></div></div></div></div></div><div id="panel-wrapper" class="col-xl-3 pl-2 text-muted"><div class="access"><div id="access-lastmod" class="post"><div class="panel-heading">Recently Updated</div><ul class="post-content pl-0 pb-1 ml-1 mt-2"><li><a href="/posts/Certificate_predict_protein_structure_using_alphafold/">(LAIDD) 알파폴드를 이용한 단백질 구조 예측 및 평가</a><li><a href="/posts/Epigenetics1/">Epigenetics I - 후성유전학의 개요</a><li><a href="/posts/Epigenetics6/">Epigenetics VI - 후성유전체 데이터 생산과 분석1</a><li><a href="/posts/AMP_guideline/">AMP Guideline</a><li><a href="/posts/R_for_Bioinformatics_4/">생물정보학을 위한 R프로그래밍 IV - 데이터 취득과 정제</a></ul></div><div id="access-tags"><div class="panel-heading">Trending Tags</div><div class="d-flex flex-wrap mt-3 mb-1 mr-3"> <a class="post-tag" href="/tags/certificate/">certificate</a> <a class="post-tag" href="/tags/bioinformatics/">bioinformatics</a> <a class="post-tag" href="/tags/bi/">BI</a> <a class="post-tag" href="/tags/python/">python</a> <a class="post-tag" href="/tags/laidd/">LAIDD</a> <a class="post-tag" href="/tags/ngs/">NGS</a> <a class="post-tag" href="/tags/statistics/">statistics</a> <a class="post-tag" href="/tags/kobic/">KOBIC</a> <a class="post-tag" href="/tags/biology/">biology</a> <a class="post-tag" href="/tags/study/">study</a></div></div></div><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@1.0.1/dist/bootstrap-toc.min.js"></script><div id="toc-wrapper" class="pl-0 pr-4 mb-5"><div class="panel-heading pl-3 pt-2 mb-2">Contents</div><nav id="toc" data-toggle="toc"></nav></div></div></div><div class="row"><div class="col-12 col-lg-11 col-xl-8"><div id="tail-wrapper" class="pl-1 pr-1 pl-sm-2 pr-sm-2 pl-md-4 pr-md-4"><div id="related-posts" class="mt-5 mb-2 mb-sm-4"><h3 class="pt-2 mt-1 mb-4 ml-1" data-toc-skip>Further Reading</h3><div class="card-deck mb-4"><div class="card"> <a href="/posts/Introduce_to_Single_Cell_Analysis/"><div class="card-body"> <em class="timeago small" data-ts="1664799040" > 2022-10-03 </em><h3 class="pt-0 mt-1 mb-3" data-toc-skip>Introduce to Single Cell Analysis</h3><div class="text-muted small"><p> 본 post는 국가생명연구자원정보센터(KOBIC) 주관 광주과학기술원(GIST) 박지환 교수님의 단일세포 분석 기술 소개를 정리한 내용입니다. Intro Single cell analysis의 다양한 기술을 자세히 알아봅니다. RNA in Mammalian Cell https://www.edwith.org/single-cell/joinLec...</p></div></div></a></div><div class="card"> <a href="/posts/Preprocessing_of_Single_Cell_Analysis_I/"><div class="card-body"> <em class="timeago small" data-ts="1665835523" > 2022-10-15 </em><h3 class="pt-0 mt-1 mb-3" data-toc-skip>Preprocessing of Single Cell Analysis I</h3><div class="text-muted small"><p> 본 post는 국가생명연구자원정보센터(KOBIC) 주관 아주대학교 의과대학 김규태 교수님의 단일세포 전사체 데이터 전분석 I를 정리한 내용입니다. Intro Single cell analysis의 의의, 구조와 형식, 전분석 과정을 이해합니다. Single Cell Analysis 의의 사람의 몸은 많은 cell이 모여 구성하고 있습니다....</p></div></div></a></div><div class="card"> <a href="/posts/Preprocessing_of_Single_Cell_Analysis_II/"><div class="card-body"> <em class="timeago small" data-ts="1665903770" > 2022-10-16 </em><h3 class="pt-0 mt-1 mb-3" data-toc-skip>Preprocessing of Single Cell Analysis II</h3><div class="text-muted small"><p> 본 post는 국가생명연구자원정보센터(KOBIC) 주관 아주대학교 의과대학 김규태 교수님의 단일세포 전사체 데이터 전분석 (실습)을 정리한 내용입니다. Intro Single cell data의 mapping, normalization, batch 제거 과정을 이해합니다. Single Cell Data Mapping Basic Data ...</p></div></div></a></div></div></div><div class="post-navigation d-flex justify-content-between"> <a href="/posts/Variants_Analysis_II/" class="btn btn-outline-primary" prompt="Older"><p>Variants Analysis II</p></a> <a href="/posts/Introduce_to_Single_Cell_Analysis/" class="btn btn-outline-primary" prompt="Newer"><p>Introduce to Single Cell Analysis</p></a></div></div></div></div><footer class="d-flex w-100 justify-content-center"><div class="d-flex justify-content-between align-items-center text-muted"><div class="footer-left"><p class="mb-0"> © 2025 <a href="https://twitter.com/username">Jonghyuk Kim</a>. <span data-toggle="tooltip" data-placement="top" title="Except where otherwise noted, the blog posts on this site are licensed under the Creative Commons Attribution 4.0 International (CC BY 4.0) License by the author.">Some rights reserved.</span></p></div><div class="footer-right"><p class="mb-0"> Powered by <a href="https://jekyllrb.com" target="_blank" rel="noopener">Jekyll</a> with <a href="https://github.com/cotes2020/jekyll-theme-chirpy" target="_blank" rel="noopener">Chirpy</a> theme.</p></div></div></footer></div><div id="search-result-wrapper" class="d-flex justify-content-center unloaded"><div class="col-12 col-sm-11 post-content"><div id="search-hints"><div id="access-tags"><div class="panel-heading">Trending Tags</div><div class="d-flex flex-wrap mt-3 mb-1 mr-3"> <a class="post-tag" href="/tags/certificate/">certificate</a> <a class="post-tag" href="/tags/bioinformatics/">bioinformatics</a> <a class="post-tag" href="/tags/bi/">BI</a> <a class="post-tag" href="/tags/python/">python</a> <a class="post-tag" href="/tags/laidd/">LAIDD</a> <a class="post-tag" href="/tags/ngs/">NGS</a> <a class="post-tag" href="/tags/statistics/">statistics</a> <a class="post-tag" href="/tags/kobic/">KOBIC</a> <a class="post-tag" href="/tags/biology/">biology</a> <a class="post-tag" href="/tags/study/">study</a></div></div></div><div id="search-results" class="d-flex flex-wrap justify-content-center text-muted mt-3"></div></div></div></div><div id="mask"></div><a id="back-to-top" href="#" aria-label="back-to-top" class="btn btn-lg btn-box-shadow" role="button"> <i class="fas fa-angle-up"></i> </a> <script src="https://cdn.jsdelivr.net/npm/simple-jekyll-search@1.10.0/dest/simple-jekyll-search.min.js"></script> <script> SimpleJekyllSearch({ searchInput: document.getElementById('search-input'), resultsContainer: document.getElementById('search-results'), json: '/assets/js/data/search.json', searchResultTemplate: '<div class="pl-1 pr-1 pl-sm-2 pr-sm-2 pl-lg-4 pr-lg-4 pl-xl-0 pr-xl-0"> <a href="{url}">{title}</a><div class="post-meta d-flex flex-column flex-sm-row text-muted mt-1 mb-1"> {categories} {tags}</div><p>{snippet}</p></div>', noResultsText: '<p class="mt-5">Oops! No result founds.</p>', templateMiddleware: function(prop, value, template) { if (prop === 'categories') { if (value === '') { return `${value}`; } else { return `<div class="mr-sm-4"><i class="far fa-folder fa-fw"></i>${value}</div>`; } } if (prop === 'tags') { if (value === '') { return `${value}`; } else { return `<div><i class="fa fa-tag fa-fw"></i>${value}</div>`; } } } }); </script> <script src="https://cdn.jsdelivr.net/combine/npm/magnific-popup@1/dist/jquery.magnific-popup.min.js,npm/lozad/dist/lozad.min.js,npm/clipboard@2/dist/clipboard.min.js"></script> <script src="https://cdn.jsdelivr.net/combine/npm/dayjs@1/dayjs.min.js,npm/dayjs@1/locale/en.min.js,npm/dayjs@1/plugin/relativeTime.min.js,npm/dayjs@1/plugin/localizedFormat.min.js"></script> <script defer src="/assets/js/dist/post.min.js"></script> <script> /* see: <https://docs.mathjax.org/en/latest/options/input/tex.html#tex-options> */ MathJax = { tex: { inlineMath: [ /* start/end delimiter pairs for in-line math */ ['$','$'], ['\\(','\\)'] ], displayMath: [ /* start/end delimiter pairs for display math */ ['$$', '$$'], ['\\[', '\\]'] ] } }; </script> <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js"> </script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4/dist/js/bootstrap.bundle.min.js"></script> <script defer src="/app.js"></script>
